ui/raidboss/data/04-sb/eureka/eureka_pyros.js

```diff
@@ -8,6 +8,24 @@
   },
   resetWhenOutOfCombat: false,
   triggers: [
+    {
+      id: 'Eureka Pyros Skoll Hoarhound Halo',
```

Comment:
I forget what these abilities are.  Are these get in or get out? Are there better instructions than just the ability names?

Reply:
those are
avoid boss back
avoid boss face

I don't know natural explanation of this in English.

Reply:
What about:
avoid boss back = `Responses.goFrontOrSides()`
avoid boss face = `Responses.getBehind()`

(the fact that I don't know what these are probably explains why I always got killed by Skoll :joy:)

Reply:
Ah, I forgot about it, reponses.

Reply:
Oh, there's no `awayFromBack()` or `goFrontOrSides()`.
Need new function.

Reply:
oh, you made new function on master branch.